staging/cert-manager-setup/templates/certificates.yaml

```diff
@@ -39,9 +39,6 @@ spec:
     - {{ . | quote -}}
   {{- end }}
 {{- end }}
-  keyAlgorithm: {{ .keyAlgorithm | default "rsa" }}
-  keyEncoding: {{ .keyEncoding | default "pkcs1" }}
-  keySize: {{ .keySize | default "2048" }}
```

Comment:
Old definition was https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec

This must be kept now with new defintion of https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificatePrivateKey

Reply:
I expect the tests must be changed for that as well.